tests/ops/integration_tests/saas/test_shopify_task.py

```diff
@@ -0,0 +1,257 @@
+import random
+
+import pytest
+
+from fidesops.ops.graph.graph import DatasetGraph
+from fidesops.ops.models.privacy_request import PrivacyRequest
+from fidesops.ops.schemas.redis_cache import PrivacyRequestIdentity
+from fidesops.ops.task import graph_task
+from tests.ops.graph.graph_test_util import assert_rows_match
+
```

Comment:
Let's add a connection test here, look at `test_datadog_task.py` for an example. I think it's good to add this so we know if failures are caused by a connectivity issue vs a deeper problem.

Reply:
I see, that's a nice test to know connectivity issue before diving into other endpoints.